components/SideNavigation.vue

```diff
@@ -213,7 +213,7 @@ export default {
     margin: 5px 16px 15px 0;
     width: 110px;
     @include lessThan($small) {
-      margin-top: 0;
+      margin: 0 16px 0 0;
```

Comment:
@bapex 細かい箇所で恐縮です。以下に修正いただけますでしょうか。
```suggestion
      margin-right: 16px;
```